app/src/js/components/Executions/overview.js

```diff
@@ -60,7 +57,7 @@ class ExecutionOverview extends React.Component {
     const { dropdowns } = collections;
     const { list } = executions;
     const { count, queriedAt } = list.meta;
-    if (list.infix && list.infix.value) {
+    if (list.asyncOperationId && list.asyncOperationId.value) {
```

Comment:
@jennyhliu or @dopeters Can you look to see why this change does or does not make sense, in terms of lint?

Reply:
So, as far as I'm concerned, this line doesn't make sense regardless. I think this is supposed to be filtering the executions based on the search term (i.e. infix) that is entered. In the executions reducer, we have 

```
[SEARCH_EXECUTIONS]: (state, action) => {
  state.list.infix = action.infix;
},
[CLEAR_EXECUTIONS_SEARCH]: (state) => {
  state.list.infix = null;
},
```
So we should be checking for `list.infix` here. However, `list.infix.value` will never be true since we're just setting a string (`action.infix` is just a string of the search query, not an object with a value param)

Basically, that line will never be true either way, so 🤷. I _think_ based on previous discussions, we may be able to just get rid of that entirely, including the `searchOperationId` function, but would be interested in @jennyhliu's input

Reply:
Yeah.  The change doesn't make sense.  We could get rid of that part.  Maybe a separate PR and also add cypress test.
Search with asyncOperationId works as the following, and infix is not used. 
executions?limit=50&page=1&sort_key%5B%5D=-createdAt&asyncOperationId=myid

Reply:
I am not sure how partial search with asyncOperationId would work.  Maybe a separate ticket to look into more and cleanup the code.

Reply:
@jennyhliu would you please write that ticket since you understand better than me?  Thank you.

Reply:
I'll revert this for the PR.

Reply:
https://bugs.earthdata.nasa.gov/browse/CUMULUS-2134

Reply:
🙌 thanks